### PR TITLE
[FEAT] Change index.html

### PR DESCRIPTION
**Pull request by mferdiyanzah**
-  modify index.html, add bblabalabal
- add unit teesting for index

Twibbon - Biofarma X MIT

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,5 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
   </head>
-  <body></body>
```
Comment (mferdiyanzah) [inline index.html:8]: kenapa dihapus cuyy

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+nyobaa
+
+.env
```
Comment (harismauladi) [inline .gitignore:3]: .env apa bang mesi

**Review by Njorommy — COMMENTED** [review]
test

**Review by harismauladi — APPROVED** [review]
sudah sesuai requirement

**Review by Njorommy — CHANGES_REQUESTED** [review]
test

**Pull request merged**